regsync/regsync/app.py

```diff
@@ -14,30 +15,28 @@ def main():
     level = logging.getLevelName(LOG_LEVEL)
     logging.basicConfig(level=level)
 
-    REGISTRY_URI = os.environ.get("APP_REGISTRY_URI")
-    if REGISTRY_URI is None:
-        logger.error("No APP_REGISTRY_URI specified.")
-        sys.exit(1)
+    MODEL_CACHE_PATH = os.environ.get("REGSYNC_MODEL_CACHE_PATH", "./models")
+    shutil.rmtree(MODEL_CACHE_PATH, ignore_errors=True)
```

Comment:
Actually, why do we delete this? If we didn't, we could preserve the cache between restarts and avoid needing to download a ton of artifacts on a cold start?